Libs/MRML/Logic/vtkMRMLSliceLayerLogic.cxx

```diff
@@ -55,7 +55,7 @@ vtkStandardNewMacro(vtkMRMLSliceLayerLogic);
 
 bool AreMatricesEqual(const vtkMatrix4x4* first, const vtkMatrix4x4* second)
 {
-  return vtkAddonMathUtilities::Matrix4x4AreEqual(first, second, /* tolerance= */ 0);
+  return vtkAddonMathUtilities::Matrix4x4AreEqual(first, second);
```

Comment:
Not sure, but before the float of the matrix where compared with -1.0, 0.0 and 1.0. I guess when comparing the elements of the two matricies there are some arrotondation errors of the order of the precision of the float.